pkg/utils/log.go

```diff
@@ -23,25 +23,29 @@ import (
 	"gopkg.in/gookit/color.v1"
 )
 
-// Log info message to stdout
+// Print output to stdout
+func Print(info string) {
+	fmt.Println(info)
+}
+
+// Print output to stdout.
+func PrintWarning(s string) {
+	fmt.Println(color.Yellow.Render("Warning:"), s)
+}
+
+// Log info message to stderr
 func Log(info string) {
-	if CanLogInfo() {
```

Comment:
This is not needed anymore?

Reply:
Since we're now logging these messages to stderr, we don't need to worry about adding noise to the output (which is on stdout).

Reply:
Gotcha, should we have these checks in `func Print(info string) {` and `func PrintWarning(s string) {` then as they are now stdout?